spec/features/idv/in_person_spec.rb

```diff
@@ -98,10 +98,18 @@
 
     # personal key page
     expect(page).to have_content(t('titles.idv.personal_key'))
-    acknowledge_and_confirm_personal_key
+    deadline = nil
+    freeze_time do
+      acknowledge_and_confirm_personal_key
+      deadline = (Time.zone.now + IdentityConfig.store.in_person_enrollment_validity_in_days.days).
+        strftime(t('time.formats.event_date'))
+    end
 
     # ready to verify page
+    enrollment_code = JSON.parse(UspsIppFixtures.request_enrollment_code_response)['enrollmentCode']
     expect(page).to have_content(t('in_person_proofing.headings.barcode'))
+    expect(page).to have_content(Idv::InPerson::EnrollmentCodeFormatter.format(enrollment_code))
+    expect(page).to have_content(t('in_person_proofing.body.barcode.deadline', deadline: deadline))
```

Comment:
Seeing a date mismatch [failure in CircleCI](https://app.circleci.com/pipelines/github/18F/identity-idp/75169/workflows/63d0be1e-fcd4-4897-a739-acbd9da5f519/jobs/133820/parallel-runs/2?filterBy=FAILED) that warrants a closer look:

>expected to find text "Your deadline to verify your identity in person is August 14, 2022." in [...] Your deadline to verify your identity in person is August 13, 2022.

May be a minor spec-specific issue with freezing time or enrollment stubbing, but will want to make sure it's not an actual issue (e.g. server timezones, etc).

Reply:
This should be behaving more consistently now as of d003779ca, but I created a separate ticket at LG-6881 to try to more accurately capture the deadline date that USPS is expecting.

Related Slack discussion: https://gsa-tts.slack.com/archives/C03FA4VBN76/p1657891564406519